From 3b84fe4375292d00ebb605a5917e66129fe5f0cb Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 24 Aug 2022 23:26:46 -0700
Subject: [PATCH] py_base_stacktrace.c: include glib.h

This file has references to g_free from glib-2.0 which needs this header

Upstream-Status: Submitted [https://github.com/abrt/satyr/pull/333]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 python/py_base_stacktrace.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/python/py_base_stacktrace.c b/python/py_base_stacktrace.c
index b9bd16c..301db84 100644
--- a/python/py_base_stacktrace.c
+++ b/python/py_base_stacktrace.c
@@ -17,7 +17,7 @@
     with this program; if not, write to the Free Software Foundation, Inc.,
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
-
+#include <glib.h>
 #include "py_common.h"
 #include "py_base_thread.h"
 #include "py_base_stacktrace.h"
-- 
2.37.2

